tests/api.c

```diff
@@ -5097,6 +5097,106 @@ static int test_wolfSSL_CertRsaPss(void)
 }
 #endif
 
+
+/* Test RSA-PSS digital signature creation and verification */
+static int test_wc_RsaPSS_DigitalSignVerify(void)
+{
+    EXPECT_DECLS;
+
+/* Early FIPS did not support PSS. */
```

Comment:
The `WC_RSA_PSS` should be enough. The older FIPS module shouldn't have that defined (I think)

Reply:
These ones are required.  Without them, tests on jenkins fail.